otter/test/rest/test_groups.py

```diff
@@ -152,8 +166,30 @@ def test_format_state_different_status(self, config_value):
             True,
             ScalingGroupStatus.ERROR,
             desired=10)
-        result = format_state_dict(state, active)
+        result = self.format(state, active)
         self.assertEqual(result['status'], 'ERROR')
+        self.assertEqual(result['errors'], [])
```

Comment:
Mostly I wanted to verify.  But yes, it's only for avoiding confusion, so someone new to the codebase who reads this won't be like "oh, it's totally valid for errors to be empty".